content/docs/insights/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Pulumi Insights
+linktitle: Pulumi Insights
+docs_home: true
+notitle: true
+menu:
+  insights:
+    identifier: insights-home
+    weight: 1
+expanded_menu_ids:
+    - insights-get-started
+    - insights-concepts
+meta_desc: Pulumi Insights provides intelligent infrastructure management with AI-powered, asset management, and compliance remediation across all your cloud resources.
```

Comment:
this seems odd: 
> AI-powered, asset management, and compliance remediation

maybe just for me

Reply:
thx; adjusted the description and meta to align to marketing pages for insights